earth_enterprise/src/common/geFilePool.cpp

```diff
@@ -49,16 +97,17 @@ bool FileReservationImpl::UnlockAndOpen_(geFilePool &pool,
                                  // our invariant numFdsUsed_ < maxNumFds
   {
     khUnlockGuard unlock(pool.mutex);
-    fd = khOpen(fname, flags, createMask);
+    pool.aFA = FileAccessorFactory::getAccessor(fname);
```

Comment:
I'm not sure about having `aFA` as a member of `geFilePool`. Each time a file is opened (via the call to `FileReservationImpl::UnlockAndOpen_`, `pool.aFA` will get reassigned. Which _may_ not cause an error, but isn't really good practice. 
I think it would be better to either:
1. Keep `aFA` as a member of the file pool and only assign it if it isn't currently assigned to anything.
2. Get rid of `aFA` in `geFilePool` and just locally use the return value of `FileAccessorFactory::getAccessor` and not keep it around.

This also gets into the topic of whether to try to re-use a single `AbstractFileAccessor` or return a new one each time. That probably warrants further discussion.